packages/react-liveness/src/components/FaceLivenessDetector/LivenessCheck/LivenessCheck.tsx

```diff
@@ -143,7 +146,7 @@ export const LivenessCheck: React.FC<LivenessCheckProps> = ({
               ? cameraMinSpecificationsHeadingText
               : cameraNotFoundHeadingText}
           </Text>
-          <Text maxWidth={300}>
+          <Text maxWidth={420}>
```

Comment:
Possible to remove a magic number by assigning `420` to a meaningfully named constant?